sources/Configuration/client/build-module/konomi-configuration.js

```
@@ -0,0 +1 @@
+var n={d:(r,o)=>{for(var t in o)n.o(o,t)&&!n.o(r,t)&&Object.defineProperty(r,t,{enumerable:!0,get:o[t]})},o:(n,r)=>Object.prototype.hasOwnProperty.call(n,r)},r={};n.d(r,{HF:()=>i,xV:()=>t,_r:()=>e});let o=null;function t(n){return null!==o||(o=function(n){try{return JSON.parse(n)}catch(n){throw new Error(`Konomi invalid configuration: ${function(n){let r="string"==typeof n?n:void 0;return r=n instanceof Error?n.message:r,null!=r?r:"unknown error"}(n)}`)}}(n),function(n){if(!Boolean(o))throw new Error("Configuration not initialized")}()),o}function i(){const n=t("");return{isDebugMode:!1,...n,iconsUrl:new URL(n.iconsUrl)}}function e(){return i()}var u=r.HF,a=r.xV,c=r._r;export{u as configuration,a as initConfiguration,c as useConfiguration};
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)